stable/concourse/README.md

```diff
@@ -103,6 +103,7 @@ The following table lists the configurable parameters of the Concourse chart and
 | `rbac.webServiceAccountName` | Name of the service account to use for web pods if `rbac.create` is `false` | `default` |
 | `rbac.workerServiceAccountName` | Name of the service account to use for workers if `rbac.create` is `false` | `default` |
 | `secrets.awsSecretsmanagerAccessKey` | AWS Access Key ID for Secrets Manager access | `nil` |
+| `secrets.teamKeys` | Array of team names and worker public keys for external workers | `nil` |
```

Comment:
(nit) The list of properties here is sorted, so it'd be nice to have new additions sorted too.